### ISSUE TITLE
Service workers only allowed on secure origins

### ISSUE THREAD

**Issue by martinleopold** [StudioProcess/rvp#174]
On http://rv.process.studio I get:
`main.js:1 DOMException: Only secure origins are allowed (see: https://goo.gl/Y0ZkNV).`

Seems like Service Workers need HTTPS. `localhost` is also a 'secure origin' so local testing should work as well:
https://www.chromium.org/blink/serviceworker/service-worker-faq

So should we just force HTTPS on rv.process.studio? I have tried it for https://rv-dev.process.studio and it seems to work just fine.

**Comment by dinony** [comment]
Yeah, I would force HTTPS. I think there is a certbot plugin for [apache](https://certbot.eff.org/docs/using.html#apache).
